src/sentry/models/file.py

```diff
@@ -249,6 +263,23 @@ def open(self):
         self.closed = False
         self.seek(0)
 
+    def _prefetch(self):
+        f = tempfile.NamedTemporaryFile()
```

Comment:
So I get that these are temp files, but it seems from my understanding, that nothing will ever delete these. This means our disks are going to fill up from temp files eventually. We should at least make a best effort to delete the temp file in the destructor or something.

Reply:
@mattrobenolt the default mode of NamedTemporaryFile is to delete. On the other PR however I have some changes on it that further improve this behavior for some edge cases.